src/Uno.UI/Microsoft/UI/Xaml/Controls/SelectorBar/SelectorBar.mux.cs

```diff
@@ -0,0 +1,222 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+// MUX Reference controls\dev\SelectorBar\SelectorBar.cpp, tag winui3/release/1.5.2, commit b91b3ce6f25c587a9e18c4e122f348f51331f18b
+
+#nullable enable
+
+using System;
+using Uno.Disposables;
+using Uno.UI.Helpers.WinUI;
+using Windows.Foundation.Collections;
+using static Uno.UI.Helpers.WinUI.CppWinRTHelpers;
+using static Microsoft.UI.Xaml.Controls._Tracing;
+
+namespace Microsoft.UI.Xaml.Controls;
+
+partial class SelectorBar
+{
+	// Change to 'true' to turn on debugging outputs in Output window
+	//bool SelectorBarTrace.s_IsDebugOutputEnabled { false };
+	//bool SelectorBarTrace.s_IsVerboseDebugOutputEnabled { false };
+
+	public SelectorBar()
+	{
+		// SELECTORBAR_TRACE_INFO(null, TRACE_MSG_METH, METH_NAME, this);
+
+		//__RP_Marker_ClassById(RuntimeProfiler.ProfId_SelectorBar);
+
+		// EnsureProperties();
+		DefaultStyleKey = typeof(SelectorBar);
+
+		var items = new ObservableVector<SelectorBarItem>();
+		SetValue(ItemsProperty, items);
+
+		Loaded += OnLoaded;
+		_loadedRevoker.Disposable = Disposable.Create(() => Loaded -= OnLoaded);
```

Comment:
Currently this doesn't do anything. This SerialDisposable isn't disposed anywhere

Reply:
I don't think it's even necessary to unsubscribe as the event publisher and subscriber are the same. An alternative is to override `OnLoaded` instead of using the Loaded event.